index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/b0efec786b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="https://orangecss.netlify.app/orange.css" />
+
+    <title>Navbar without JS</title>
+  </head>
+  <body>
+    <header class="nav-header">
+      <h2 class="nav-logo">Logo</h2>
+      <input id="nav-toggle" type="checkbox" class="nav-toggle" />
+      <label class="nav-toggle-label" for="nav-toggle"
+        ><span><i class="fas fa-bars"></i></span
+      ></label>
+      <nav class="navbar">
+        <ul class="nav-items">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```

Comment:
Hey Vinay, your navbar code is easily understandable and well-written follow the same in other projects also.

Reply:
Yeah okay Aravindan